public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -12,9 +16,11 @@ function cc_scripts() {
 
 if ( ! function_exists( 'cc_styles' ) ) {
     function cc_styles() {  
-        wp_enqueue_style('cc-bootstrap', get_template_directory_uri().'/dist/lib/css/bootstrap.min.css');
-        wp_enqueue_style('style', get_template_directory_uri().'/style.css');
-
+       
+        wp_enqueue_style('cc-custom-bsi','https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css');      
```

Comment:
what does bsi stands for? I cannot get a clear meaning out of it if used somewhere else. Please change the name. Make it simpler and descriptive.